test/integration/master_routes_test.go

```diff
@@ -183,17 +188,79 @@ func TestRootRedirect(t *testing.T) {
 		t.Fatalf("Unexpected index: \ngot=%v,\n\n expected=%v,\n\ndiff=%v", got.Paths, expectedIndex, diff.ObjectDiff(expectedIndex, got.Paths))
 	}
 
-	req, err = http.NewRequest("GET", masterConfig.OAuthConfig.MasterPublicURL, nil)
-	req.Header.Set("Accept", "text/html")
-	resp, err = transport.RoundTrip(req)
+	// Create fake config map to test console redirect
```

Comment:
I'd accept a simple `apiVersion` literal.

Reply:
Sorry, I do not understand the context of this comment